### PR TITLE
Fix in ImageNet_txt

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Summary

- Added an extension to the image name in the annotation file (this is done in AC)

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by zhiltsov-max — COMMENTED** [review]
It doesn't affect the image search code? Is it needed after?

**Comment by yasakova-anastasia** [comment]
It doesn't affect the image search code.

File: datumaro/plugins/imagenet_txt_format.py [hunk 1]
```diff
@@ -62,13 +62,13 @@ def _load_items(self, path):
...
                         item_id = item[1]
-                        label_ids = [int(id) for id in item[2].split()]
+                        label_ids = [int(id) for id in item[2].split()[1:]]
                     else:
                         raise Exception("Line %s: unexpected number "
                             "of quotes in filename" % line)
                 else:
                     item = line.split()
-                    item_id = item[0]
+                    item_id = osp.splitext(item[0])[0]
```
Comment (zhiltsov-max) [inline datumaro/plugins/imagenet_txt_format.py:71]: I don't think we can allow to match any extension, if it is specified.

**Comment by zhiltsov-max** [comment]
Do we need image search at all after these changes?

**Comment by yasakova-anastasia** [comment]
> Do we need image search at all after these changes?

Image search isn't needed, removed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `ItemTransform` class, which describes item-wise dataset `Transform`s (<https://github.com/openvinotoolkit/datumaro/pull/297>)
 - `keep-empty` export parameter in VOC format (<https://github.com/openvinotoolkit/datumaro/pull/297>)
 - A base class for dataset validation plugins (<https://github.com/openvinotoolkit/datumaro/pull/299>)
+- Extensions for images in Imagenet_txt annotation file (<https://github.com/openvinotoolkit/datumaro/pull/302>)
 
 ### Changed
 - Tensorflow AVX check is made optional in API and is disabled by default (<https://github.com/openvinotoolkit/datumaro/pull/305>)
```
Comment (zhiltsov-max) [inline CHANGELOG.md:21]:
```suggestion
- Tensorflow AVX check is made optional in API and is disabled by default (<https://github.com/openvinotoolkit/datumaro/pull/305>)
- Extensions for images in ImageNet_txt are now mandatory (<https://github.com/openvinotoolkit/datumaro/pull/302>)
```

**Review by zhiltsov-max — APPROVED** [review]

**Pull request merged**